projects/kserve/visualizations/kserve-llm/plotting/errors.py

```diff
@@ -169,7 +182,10 @@ def generateFinishReasonData(entries, variables):
             #elif reason == 1: # MAX_TOKENS: good, ignore
             #    continue
             else:
-                reason = STOP_REASONS[reason]
+                if isinstance(reason, str): # vLLM
```

Comment:
I think you should try to make that more reliable, eg you can lookup something in the TOPSAIL configuration to detect vllm, or look it up in the pod spec or elsewhere in the artifacts